scripts/git-hooks/post-commit

```diff
@@ -1,18 +1,14 @@
 #!/usr/bin/env bash
 
 # That's a sample standard post-commit hook that checks if a current branch is already managed by git machete,
-# and if it isn't, suggests to add it onto its inferred parent.
+# and if it isn't, suggests to add it onto its inferred upstream.
 
 # Don't do anything if we're in a detached HEAD state.
-git symbolic-ref --quiet HEAD >/dev/null || exit 0
+git machete show current >/dev/null 2>/dev/null || exit 0
```

Comment:
Unrel, just updated to match https://github.com/VirtusLab/git-machete/blob/master/hook_samples/post-commit